faq.html

```diff
@@ -155,10 +155,13 @@ <h2 id="general">General</h2>
       <a href="{{page.root}}/img/software-carpentry-banner.png">Software Carpentry logo</a>
       are both trademarked,
       and may only be used with our explicit prior permission.
-      We normally grant this to any training course that covers our
-      <a href="#core-topics">core topics</a>
-      has at least one <a href="{{page.root}}/pages/team.html">certified instructor</a> teaching,
-      and sends us summary information about attendees.
+      We normally grant this to any training course that:
```

Comment:
On Tue, Mar 17, 2015 at 07:12:21AM -0700, Aleksandra Pawlik wrote:
> They have to be compliant with some specifications. Obviously, it's
> unlikely someone will create their own version of the template…

Making this explicit would help, but do we *need* to have a homepage
before something can be a SWC-branded event?  In amy, the Event.url
field is blank and nullable [1], and it's been nullable since the
field landed in swcarpentry/amy@e75538a5 (Displaying repositories for
workshops, 2014-12-01) as Event.repo.  Personally, I *like* it when
events have a public homepage, but I don't think we should require it
before allowing them to use SWC branding.

[1]: https://github.com/swcarpentry/amy/blob/e50c395ef82ef7d9481a63e8194a3777ccf0d834/workshops/models.py#L259